drivers/gles3/shaders/copy.glsl

```diff
@@ -169,7 +169,7 @@ void main() {
 	// The Matrix[2][0] (= asym_proj.x) and Matrix[2][1] (= asym_proj.z) values are what provide the right shift in the image.
 
 	vec3 cube_normal;
-	cube_normal.z = -1000000.0;
+	cube_normal.z = -1.0;
```

Comment:
This change obsoletes the above comments, so they should be edited too.

Reply:
Yes; I pushed a proposal for the comment change.